misago/users/signals.py

```diff
@@ -1,11 +1,33 @@
+from datetime import timedelta
+
+from django.contrib.auth import get_user_model
+from django.db.models import Q
 from django.dispatch import Signal, receiver
+from django.utils import timezone
+
+from misago.conf import settings
+from misago.core.utils import ANONYMOUS_IP
+
 
+UserModel = get_user_model()
 
 anonymize_user_content = Signal()
 delete_user_content = Signal()
 username_changed = Signal()
+remove_old_ips = Signal()
```

Comment:
`remove_old_ips = Signal()` should come before the `username_changed = Signal()`